python/lsst/daf/persistence/repositoryCfg.py

```diff
@@ -317,4 +317,15 @@ def __repr__(self):
             self._policy)
 
 
-yaml.add_constructor(u"!RepositoryCfg_v1", RepositoryCfg.v1Constructor)
+Loaders = [yaml.Loader, ]
```

Comment:
Maybe `loaderList` rather than `Loaders`?  Otherwise, it looks like a class name.